automation.py

```diff
@@ -90,6 +92,7 @@ def handle_dupe_files(dir, file):
         new_file_name = begins_with + "_" + str(count) + file_extension
     return new_file_name
 
+
 if __name__ == "__main__":
```

Comment:
can you clean up the main function.  take out all the comments and put in just the functions that need to be called.

if __name__ == "__main__":
    create_dir(folders_to_create)
    files = get_files(root_dir)
    move_files(files)

Reply:
Pushed those changes. Should be good to go